### PR TITLE
Added actions to split-upload/download artifacts of large size

### PR DESCRIPTION
**Pull request by jithinjk**
**Description:**
- Actions to perform split and upload artifacts that are greater than the default upload limit(500MB) of CDS.
- CDS_SplitUploadArtifact: Uses split command to split artifacts and upload them using worker.
- Above action can uploads the MD5checksum of the uploading file.
- CDS_SplitDownloadArtifact: Downloads the artifacts and performs checksum validation after downloading.

PFA screenshots:

@ovh/cds @yesnault @fsamin

### CODE REVIEW

File: contrib/actions/cds-split-upload.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    description:    Prefix to be added to destination files after split. (Mandatory)
+  tag:
+    type:           string
+    description:    Tag to identify uploaded artifacts. Default tag value is CDS run version number. (Optional)
+  splitSize:
+    type:           string
+    default:        "200m"
+    description:    Size of each split files. Default size is 200MB. (Optional)
+requirements:
+  - binary:         bash
```
Comment (yesnault) [inline contrib/actions/cds-split-upload.yaml]: need split binary

**Comment by yesnault** [comment]
can you rename *.yaml -> *.yml please? 

FYI: this PR https://github.com/ovh/cds/pull/2467 rename extension for existing actions.

File: contrib/actions/cds-split-download.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  - ""
+  - 'worker download --tag=${TAG} --pattern="${PATTERN}"'
+  - ""
+  - 'cat ${PREFIX_HANDLE}-* > ${DESTINATION}'
+  - 'y=$(md5sum ${DESTINATION} | cut -d " " -f 1)'
+  - ""
+  - if grep -Fxq "${y}" *-md5 ; then
+  - "\techo -e \"\nDownloaded Artifacts Are Matching...\""
+  - else
+  - "\techo -e \"\nDownloaded Artifacts Are Not Matching...\n\""
```
Comment (blaryjp) [inline contrib/actions/cds-split-download.yml]: no exit 1 ?

Comment (jithinjk) [reply contrib/actions/cds-split-download.yml]: @blaryjp Thanks for pointing that out. I'll add that.

**Pull request merged**